File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        lis = quickSort(nums)
        for i in range(len(nums)):
            nums[i]=lis[i]
def quickSort(nums):
    if len(nums)<=1:
        return nums
    pivot = nums[-1]
    left = []
    right=[]
    for i in range(0,len(nums)-1):
        if nums[i] <= pivot:
            left.append(nums[i])
        else:
            right.append(nums[i])
    return quickSort(left)+[pivot]+quickSort(right)
        

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] arrList = new int[nums.length];
        int right =1;
        int left =1;
        for(int i =0; i<nums.length; i++){
            arrList[i]=left;
            left = left*nums[i];
        }
        for(int i=nums.length-1; i>=0; i--){
            arrList[i]=arrList[i]*right;
            right = right *nums[i];
        }
        return arrList;
    }
    
}  
